app/controllers/application_controller.rb

```diff
@@ -1060,11 +1059,7 @@ def get_reports_menu(group = current_group, tree_type = "reports", mode = "menu"
   # Render the view data to xml for the grid view
   def view_to_xml(view, from_idx = 0, to_idx = -1, options = {})
     # Get the time zone in effect for this view
-    if view.db.downcase == 'miqschedule'
-      tz = MiqServer.my_server.get_config("vmdb").config.fetch_path(:server, :timezone) || "UTC"
-    else
-      tz = Time.zone
-    end
+    tz = view.db.downcase == 'miqschedule' ? server_timezone : Time.zone
```

Comment:
do you mind wrapping the equality check into parens `(view.db.downcase == 'miqschedule')` for better readability?